src/Microsoft.Health.Fhir.Liquid.Converter.FunctionalTests/Microsoft.Health.Fhir.Liquid.Converter.FunctionalTests.csproj

```diff
@@ -8,6 +8,8 @@
   
   <ItemGroup>
     <PackageReference Include="Microsoft.NET.Test.Sdk" Version="16.7.1" />
+    <PackageReference Include="nHapi" Version="2.5.0.6" />
+    <PackageReference Include="System.Configuration.ConfigurationManager" Version="5.0.0" />
```

Comment:
Are we using this package in codes?

Reply:
Yes, we use `nHapi` to validate V2 messages, and the `ConfigurationManager` is a dependency of `nHapi`.